File: LeetCode-Programs/Baseball Game.py
class Solution:
    def calPoints(self, ops: List[str]) -> int:
        new_list=[]
        for i in ops:
            if i=="+":
                new_list.append(new_list[len(new_list)-1]+new_list[len(new_list)-2])
            elif i=="D":
                new_list.append(2*new_list[len(new_list)-1])
            elif i=="C":
                new_list.pop()
            else:
                new_list.append(int(i))
        return sum(new_list)

File: LeetCode-Programs/Baseball Game.cpp
class Solution {
public:
    int calPoints(vector<string>& ops) {
        vector<int> vec;
        for(int i=0; i<ops.size(); i++){
            if(ops[i]=="+")
                vec.push_back(vec[vec.size()-1]+vec[vec.size()-2]);
            else if(ops[i]=="D")
                vec.push_back(vec[vec.size()-1]*2);
            else if(ops[i]=="C")
                vec.pop_back();
            else
                vec.push_back(stoi(ops[i]));
        }
        return accumulate(vec.begin(), vec.end(), 0);
        
    }
};
